imsim/stamp.py

```diff
@@ -494,7 +499,11 @@ def fix_seds(prof):
                                method='fft',
                                offset=fft_offset,
                                image=fft_image,
-                               wcs=wcs)
+                               wcs=wcs,
+                               photon_ops=photon_ops,
+                               sensor=galsim.Sensor(),
```

Comment:
This won't be necessary once GalSim 2.4.6 is released.  I fixed the test that had been triggering the assert failure.

Reply:
One other thing here.  The photon sampling of the FFT image uses an rng, so you should also pass `rng=self.rng` here.